libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1753,9 +1755,11 @@ bool AP_Mission::mission_cmd_to_mavlink_int(const AP_Mission::Mission_Command& c
         packet.param3 = cmd.content.do_engine_control.height_delay_cm*0.01f;
         break;
 
+#if AP_MISSION_NAV_PAYLOAD_PLACE_ENABLED
     case MAV_CMD_NAV_PAYLOAD_PLACE:
         packet.param1 = cmd.p1/100.0f; // copy max-descend parameter (cm->m)
```

Comment:
Driveby, but can you use a *0.01f here instead?

Reply:
I'd rather not complicate the PR... especially as this is not exactly a "hot" path :-)

I'll be working more in this area, so will slip that change in in a future PR :-)